cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -368,6 +368,129 @@ objects:
         }
       }
 
+# TODO: parameterize data.platform-identities.yaml after templatize refactor and helm chart conversion
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: azure-operators-managed-identities-config
+    namespace: ${NAMESPACE}
+  data:
+    azure-operators-managed-identities-config.yaml: |
+      controlPlaneOperatorsIdentities:
+
+        cloud-controller-manager:
+          minOpenShiftVersion: 4.17
+          maxOpenShiftVersion: 4.17
+          azureRoleDefinitionResourceId: '/subscriptions/1d3378d3-5a3f-4712-85a1-2485495dfc4b/providers/Microsoft.Authorization/roleDefinitions/ebe170ec-1247-536a-86d9-74c829dd9844'
```

Comment:
I noticed that the roleDefinition is scoped to our subscription, I am wondering if this config as is can be  re-usable in say INT, Stage or Prod env? @tony-schndr @bennerv do you know or we would need to update the value across the different envs?

Reply:
INT and PROD will use different roles.  I am creating Custom roles in dev that are hardcoded in the repo for now.  I will be converting cluster-service deployment to use helm in the near future, apart of that change the config map will be able to use a different data.
I took this route for 2 reasons:
1. To unblock cluster-service work.
2. To prevent rework trying to make the config map data configurable in oc templates, since deployment of cluster-service is moving to helm charts.